Components/Card/card.css

```diff
@@ -0,0 +1,70 @@
+.card-container {
+  display: flex;
+  align-items: center;
+  gap: 1.875rem;
+  padding: 0.625rem;
+}
+
+.card {
+  position: relative;
+  border: none;
+}
+
+.card-img {
+  
+}
```

Comment:
Can remove this if there there is no styling here.

Reply:
corrected it